contracts/modules/capital/Pool.sol

```diff
@@ -699,11 +684,6 @@ contract Pool is IPool, MasterAware, ReentrancyGuard {
   }
 
   function updateUintParameters(bytes8 code, uint value) external onlyGovernance {
-    if (code == "MIN_ETH") {
-      minPoolEth = value;
-      return;
-    }
-
```

Comment:
Remove the function entirely?

Reply:
We have a Governance proposal for this i would keep it in case a parameter comes up